openapi-python-templates/model.mustache

```diff
@@ -1,20 +1,22 @@
+from __future__ import annotations
+
 from datetime import datetime
-from typing import Optional, List
+from typing import Optional, List, Dict
 from typing import Any  # noqa
 from typing_extensions import Literal
 from uuid import UUID
-from pydantic import BaseModel, Schema
+from pydantic import BaseModel, Field
 
 {{#models}}
 {{#model}}
 class {{classname}}(BaseModel):
 {{#vars}}
 {{#isEnum}}
-    {{name}}: 'Literal[{{#allowableValues}}{{#values}}"{{{this}}}"{{^-last}}, {{/-last}}{{/values}}{{/allowableValues}}]' = Schema({{#required}}...{{/required}}{{^required}}None{{/required}}, alias="{{baseName}}")  # type: ignore
+    {{name}}: Literal[{{#allowableValues}}{{#values}}"{{{this}}}"{{^-last}}, {{/-last}}{{/values}}{{/allowableValues}}] = Field({{#required}}...{{/required}}{{^required}}None{{/required}}, alias="{{baseName}}")  # type: inore
 {{/isEnum}}
 {{^isEnum}}
-    {{name}}: "{{^required}}Optional[{{/required}}{{>_dataTypeModel}}{{^required}}]{{/required}}" = Schema({{#required}}...{{/required}}{{^required}}None{{/required}}, alias="{{baseName}}")  # type: ignore
+    {{name}}: {{^required}}Optional[{{/required}}{{>_dataTypeModel}}{{^required}}]{{/required}} = Field({{#required}}...{{/required}}{{^required}}None{{/required}}, alias="{{baseName}}")  # type: ignore
```

Comment:
I think that maybe I should revert this change... it works well with python 3.7+ because of the support of deferred annotations, but this cannot be imported in python 3.6 for example, which makes the client incompatible with anything below 3.7... What do you think?

Reply:
I think we should keep with the `"`s for now; compatibility with 3.6 is still nice to offer, and you need them either way if you don't add `from __future__ import annotations` at the top.